python/paddle/v2/framework/tests/test_save_restore_op.py

```diff
@@ -1,71 +0,0 @@
-import paddle.v2.framework.core as core
-import paddle.v2.framework.framework as framework
-import paddle.v2.framework.executor as executor
-
-import numpy as np
-import unittest
-import os
-import sys
-import shutil
-
-FOLDER_PATH = "./tmp_test_dir"
-
-
-class TestSaveRestoreOp(unittest.TestCase):
```

Comment:
I think we need the python side of test. Because we need to expose the `Save Op` as a python API.

Reply:
It will be done along with @Canpio 's PR.